set.mm

```diff
@@ -439147,6 +439147,18 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
 htmldef "RRhat" as "RRhat";
   althtmldef "RRhat" as "&#8477;&#x302;";
   latexdef "RRhat" as "\hat{\mathbb{R}}";
+htmldef "NNbar" as "NNbar";
+  althtmldef "NNbar" as "&#8469;&#x305;";
+  latexdef "NNbar" as "\bar{\mathbb{N}}";
+htmldef "ZZbar" as "ZZbar";
+  althtmldef "ZZbar" as "&#8484;&#x305;";
+  latexdef "ZZbar" as "\bar{\mathbb{Z}}";
+htmldef "ZZhat" as "ZZhat";
+  althtmldef "ZZhat" as "&#8484;&#x302;";
+  latexdef "ZZhat" as "\hat{\mathbb{Z}}";
+htmldef "||C" as " ||C ";
+  althtmldef "||C" as " &#8741;<sub>&#8450;</sub> ";
```

Comment:
What does the C stand for?

Reply:
I needed to distinguish it from the already existing token `||` so I chose "C" for "complex".

Reply:
(added a reference in the comment of df-bj-divc)